apps/condo/domains/miniapp/schema/fields/accessRight.js

```diff
@@ -0,0 +1,27 @@
+const { Relationship } = require('@keystonejs/fields')
+const { getById } = require('@core/keystone/schema')
+const { SERVICE } = require('@condo/domains/user/constants/common')
+const { NON_SERVICE_USER_ERROR } = require('@condo/domains/miniapp/constants')
+
+const SERVICE_USER_FIELD = {
+    schemaDoc: `Link to user. Note, that user must be of "${SERVICE}" type`,
+    type: Relationship,
+    ref: 'User',
+    isRequired: true,
+    knexOptions: { isNotNullable: true }, // Required relationship only!
+    kmigratorOptions: { null: false, on_delete: 'models.PROTECT' },
+    hooks: {
+        validateInput: async ({ resolvedData, fieldPath, addFieldValidationError }) => {
```

Comment:
can we add the same check for other apps?

Reply:
It's already integrated with B2B Apps as well...

Will add it to billing / acquiring too

Reply:
done